package.json

```diff
@@ -52,7 +52,7 @@
     "start:client": "react-scripts start",
     "start:firebase": "firebase emulators:start --import=./emulators.backup",
     "start:storybook": "start-storybook -p 9009 -s public --quiet",
-    "prepare": "husky install"
```

Comment:
This gets executed on cloud build, but they enforce `NODE_ENV=production` which skips installing husky. Is it better to run this command conditionally or move husky to regular dependencies?

Reply:
I think it's better to run it conditionally